developer/src/kmc-kmn/test/kmw/util.ts

```diff
@@ -1,13 +1,16 @@
 
-interface ETLResult {
+export interface ETLResult {
   js: string;
   touchLayout: string;
 }
 
 export function extractTouchLayout(js: string): ETLResult|null  {
   let m = /KVKL=(?<kvkl>.+?);[\r\n]/ds.exec(js);
   if(!m) {
-    return null;
+    return {
+      js: js,
+      touchLayout: 'null'
+    };
```

Comment:
we always want to compare the js, even if no touch layout is present in the compiled keyboard.